### ISSUE TITLE
[Feature Request]: Dynamic Fit Indices

### ISSUE THREAD

**Issue by kyle-hickerson** [jasp-stats/jasp-issues#2608]
### Description

Addition of Dynamic Fit Indices from the dynamic package.

### Purpose

Allows for better assessment of model fit.

### Use-case

Test for differing levels of model misspecification.

### Is your feature request related to a problem?

No.

### Is your feature request related to a JASP module?

SEM

### Describe the solution you would like

Implemention of functionality from the dynamic package.

### Describe alternatives that you have considered

NA

**Comment by tomtomme** [comment]
@khh200 
This seems to be a duplicate of https://github.com/jasp-stats/jasp-issues/issues/2083
That one discusses implementing R dynamic for CFA. But since CFA can be done via SEM, I wonder if we can close this one as a duplicate and discuss the implementation details over at #2083

Do you agree?

**Comment by shun2wang** [comment]
Yes, I agree with @juliuspfadt , we don't need to merge issues, each issue/module needs to be dealt with individually for developers.
